Dojo/test/ConcessionariaTest.java

```diff
@@ -0,0 +1,18 @@
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class ConcessionariaTest {
+
+    @Test
+    void ConcessionariaTest(){
+         double Saldo =0;
+
+        Assertions.assertEquals(00,00);
+    }
+    @Test
+    void VendaCarro() {
+        Carro carro = new Carro("Corolla", 150.00);
```

Comment:
O `.` é casa decimal. O valor que está sendo colocado aqui é de R$ 150,00 em vez de R$ 150.000,00.
```suggestion
        Carro carro = new Carro("Corolla", 150000.00);
```